### ISSUE TITLE
[ISSUE]: DVD9 games are installed incorrectly

### ISSUE THREAD

**Issue by BloodRaynare** [ps2homebrew/hdl-dump#62]
### Checks

- [X] I have checked existing issues for duplicates and found none

### Describe the issue

Latest development build of hdl_dump broke DVD9 games installation (probably miscalculated layer break information) causing freezes or glitches. 
Tested with "God of War" (SCUS-97399) (Note that I didn't compressed it to ZSO, just plain ISO). In GOW1's case, loading my save file which I've saved in "Cliffs of Madness" area freezes the game during the loading screen after the save has been loaded.
Installing it with old rev. 47 25/12/2015 build works fine.

### Console model

SCPH-50000

**Comment by israpps** [comment]
Is it a md5 verified image?

**Comment by BloodRaynare** [comment]
Yes, MD5 hash (and the CRC32) was confirmed, same as the redump ones:

CRC32: 587635E4
MD5: 7DD68E0F5964ABCF8282FC4E1AD41400

**Comment by AcidReach** [comment]
What is the filesystem format of the disk you are installing isos from?
I was having the same issue and I realised it was the HDD filesystem format I was using to store my isos 
I use Linux so my external HDD was in EXT4 format 
I tried installing the games using hdl_dump using both Linux, and windows with the extfsd driver. Neither worked.
I then copied GOW iso to an exfat external HDD and used hdl_dump on windows to install the game to my ps2 HDD and tested by loading up my save which is on the second layer and it works perfectly.

**Comment by BloodRaynare** [comment]
> What is the filesystem format of the disk you are installing isos from? I was having the same issue and I realised it was the HDD filesystem format I was using to store my isos I use Linux so my external HDD was in EXT4 format I tried installing the games using hdl_dump using both Linux, and windows with the extfsd driver. Neither worked. I then copied GOW iso to an exfat external HDD and used hdl_dump on windows to install the game to my ps2 HDD and tested by loading up my save which is on the second layer and it works perfectly.

I'm using Ext HDD and was formatted in NTFS

**Comment by 10522** [comment]
@BloodRaynare, try [latest](https://github.com/ps2homebrew/hdl-dump/releases/tag/latest) Development build.

**Comment by jqian33** [comment]
> [@BloodRaynare](https://github.com/BloodRaynare), try [latest](https://github.com/ps2homebrew/hdl-dump/releases/tag/latest) Development build.

I can confirm that [this build](https://github.com/ps2homebrew/hdl-dump/releases/tag/latest) fixed my dual layer iso image loading issue. When I installed God of War II using the latest stable rev.47, I am unable to load any saves past "Crossing the lowlands" section which is on the second layer. After I installed GoW 2 using the aboved mentioned build, I can load my saves.

**Comment by israpps** [comment]
Here's a most likely dumb idea, but worth trying if you haven't already 

Try compatibility mode 5...

**Comment by 10522** [comment]
@AKuHAK @rickgaiser @israpps @uyjulian @fjtrujy @BloodRaynare please close this issue as it has been fixed.
